src/examples/wasm/Makefile

```diff
@@ -0,0 +1,14 @@
+export: clean wasm_exec
+	tinygo build -o ./html/wasm.wasm -target wasm ./export/wasm.go
+	cp ./export/wasm.js ./html/
+	cp ./export/index.html ./html/
+
+main: clean wasm_exec
+	tinygo build -o ./html/wasm.wasm -target wasm ./main/main.go
+	cp ./main/index.html ./html/
+
+wasm_exec:
+	cp ../../../targets/wasm_exec.js ./html/
+
+clean:
```

Comment:
Perhaps the clean task should recreate the empty html directory? Otherwise you get the error 

```shell
$ make export
rm -rf ./html/*
cp ../../../targets/wasm_exec.js ./html/
cp: cannot create regular file './html/': Not a directory
Makefile:11: recipe for target 'wasm_exec' failed
make: *** [wasm_exec] Error 1
```

Reply:
Good idea, I'd only tested it with an existing folder.

Reply:
Fixed

Reply:
Thank you!